Browser-storage.js

```diff
@@ -1,7 +1,7 @@
 const contactForm = document.querySelector('#signup');
-const content = JSON.parse(window.localStorage.getItem('storage'));
+const content = JSON.parse(window.localStorage.getItem('datastorage'));
 function check() {
```

Comment:
- [ ] Please fix this console error, I can not mark the code here because this error is coming from your Card-model.js file

![image](https://user-images.githubusercontent.com/83803414/157841721-c343df40-1e8d-45f5-ad95-53b13e924159.png)

Reply:
I have resolved it from my code in card-js file.